config/schema/document_types/vehicle_recalls_and_faults_alert.json

```diff
@@ -0,0 +1,111 @@
+{
+  "fields": [
+    "fault_type",
+    "alert_issue_date",
+    "item_type",
+    "manufacturer",
+    "item_model",
+    "serial_number",
+    "build_start_date",
+    "build_end_date"
+  ],
+
+  "allowed_values": {
+    "fault_type": [
+      {
+        "label": "Recall",
+        "value": "recall"
+      },
+      {
+        "label": "Non-urgent fault",
+        "value": "non_urgent_fault"
+      }
+    ],
+
+    "item_type": [
+      {
+        "label": "Vehicle",
+        "value": "vehicle"
+      },
+      {
+        "label": "Baby seat",
+        "value": "baby-seat"
+      },
+      {
+        "label": "Tyres",
+        "value": "tyres"
+      },
+      {
+        "label": "Parts",
+        "value": "parts"
+      },
+      {
+        "label": "Agricultural equipment",
+        "value": "agricultural-equipment"
+      },
+      {
+        "label": "Other accessories",
```

Comment:
For item types the list should serve the purpose well and not change often if at all.

However, we will definitely need to add more manufacturers to the list (you can see on http://www.dft.gov.uk/vosa/apps/recalls/ there are more manufacturers). We could allow any value for the manufacturers field, but we still need:

1) to have a dropdown with predefined values in specialist-publisher. This will ensure there is no data duplication, typos etc.
2) to have a facet for filtering and subscribing to a specific manufacturer email alerts. 

I don't know where the finder frontend fetches facets from, is it rummager? In any case we would like to test-drive this asap and once we're convinced finders is the right solution to this problem, we can solve it long-term.

I also don't know if changing the schema in rummager is difficult. If so, let me know.